packages/grafana-data/package.json

```diff
@@ -56,25 +60,18 @@
     "@types/node": "16.11.6",
     "@types/papaparse": "5.2.0",
     "@types/react": "17.0.30",
-    "@types/rollup-plugin-visualizer": "4.2.1",
```

Comment:
i think @jackw has been using this for bundle size and insights?

Reply:
thanks @leeoniya 👍 
can't find anywhere in the code that it's currently being used.
if it's an ad-hoc local dev type thing, that sounds more suited to being a global package install anyway i think? 🤔

Reply:
Thanks for spotting this. Yeah I use this from time to time to better understand what's in the output when making changes to the packages. I think it's fine to remove it for now and next time I need it I'll figure something out.